phpBB/phpbb/log/log.php

```diff
@@ -694,6 +694,16 @@ public function get_logs($mode, $count_logs = true, $limit = 0, $offset = 0, $fo
 						$log_data_ary = array_merge($log_data_ary, array_fill(0, $num_args - count($log_data_ary), ''));
 					}
 
+					if (stripos($row['log_operation'], 'BAN_LOG_') === 0)
```

Comment:
I think you can just use `strpos` here as it can be case sensitive.
And from just looking at the PR, shouldn't it be `LOG_BAN_`?

Reply:
Correct. Fixed in subsequent commit.